codegen/config/checkstyle/checkstyle.xml

```diff
@@ -32,7 +22,7 @@
     <!-- Files must contain a copyright header. -->
     <module name="RegexpHeader">
         <property name="header"
-                  value="/\*\n \* Copyright 20(19|20|21|22) Amazon\.com, Inc\. or its affiliates\. All Rights Reserved\.\n"/>
+                  value="/\*\n \* Copyright( 20(19|20|21|22)|) Amazon\.com, Inc\. or its affiliates\. All Rights Reserved\.\n"/>
```

Comment:
should this include 23 now?

Reply:
My intent was to have the short-form used starting now. There aren't any `Copyright 2023` dates in current use, so we can switch now and not need to allow those.